src/components/v5/common/Navigation/consts.ts

```diff
@@ -0,0 +1,7 @@
+export const sidebarButtonStyles =
+  'hover:!bg-gray-800 !justify-start !gap-3 !px-2 !py-1 !text-base-white w-full';
+
+export const sidebarButtonIconStyles =
+  'aspect-auto h-5 w-auto flex-shrink-0 text-base-white';
+
+export const sidebarButtonTextStyles = 'text-md font-medium text-base-white';
```

Comment:
Can you use the tw`` utility function as the `Class` suffix for these? And put in a `.styles.ts` file. See #2045 for more info.

(And any other similar class exports)

Reply:
Oh fair enough, I shall do that 👌